app/models/ability.rb

```diff
@@ -8,7 +8,10 @@ def initialize(user)
       if user.admin?
         can :manage, :all
       else
-        can :manage, :all
+        can :read, JobOpening
+        can :manage, Submission, user_id: user.id
```

Comment:
This, too.  Now users can only `manage` Submissions that belong to them.  They won't be able to touch any other user's submissions at all.